MushroomObserver/mushroom-observer
# Block damaging mergers

Prevents users from making a merger where the Name being edited is:
- An Approved Synonym of a Proposed Name
- A correctly spelled Ancestor of a Proposed Name

#### Manual Test Script
- Create Name 1
- Create Name 2 
- Propose Name 2 for any Observation
- Deprecate Name 2 in favor of Name 1
- Edit Name 1 to be the same as an existing Name
**Expected Result**: Names are not merged; You are sent to `email_merge_request`
- For any Observation, Propose a Name that's a new species in a new genus, e.g. `Anabanana fontana`
- Edit the new genus (Ex: `Anabanana`) to be the same as an existing genus, e.g., `Abies`
**Expected Result**: same as above

Delivers https://www.pivotaltracker.com/story/show/171308819
